crates/hyperqueue/src/dashboard/ui/screens/worker/cpu_util_table.rs

```diff
@@ -56,10 +57,14 @@ pub fn render_cpu_util_table(
     ));
     let body_block = styles::table_block_with_title(title);
 
+    let style = match is_selected {
```

Comment:
Rather than sending a bool to the function, we should send the style. And create methods like `table_style_selected()`, `table_style_deselected()` etc.